android.patch

```diff
@@ -0,0 +1,25 @@
+From 656be483d6053db7dcb8f83d4c79c68dc732c1d5 Mon Sep 17 00:00:00 2001
+From: Gary Mathews <gmathews@appcelerator.com>
+Date: Sat, 23 Mar 2019 09:57:46 -0700
+Subject: [PATCH] fix android stdout
+
+---
+ infra/mb/mb_config.pyl | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/infra/mb/mb_config.pyl b/infra/mb/mb_config.pyl
+index 8a6cba71e0..d2004c086e 100644
+--- a/infra/mb/mb_config.pyl
++++ b/infra/mb/mb_config.pyl
+@@ -569,7 +569,7 @@
+ 
+   'mixins': {
+     'android': {
+-      'gn_args': 'target_os="android" v8_android_log_stdout=true',
++      'gn_args': 'target_os="android" v8_android_log_stdout=false',
```

Comment:
Set `v8_android_log_stdout` to `false` as we want to output using `__android_log`